Index: stylesheet.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>button {\n    display: inline-block;\n    padding: 15px 25px;\n    font-size: 24px;\n    cursor: pointer;\n    text-align: center;\n    text-decoration: none;\n    outline: none;\n    color: #fff;\n    background-color: indianred;\n    border: none;\n    border-radius: 15px;\n    box-shadow: 0 9px #999;\n}\n\nbutton:hover {background-color: darkred}\n\nbutton:active {\n    background-color: darkred;\n    box-shadow: 0 5px #666;\n    transform: translateY(4px);\n}\n.container {\n    display: grid;\n    grid-template-columns: 400px 600px 400px;\n    text-align: center;\n\n\n}\n.column {\n    padding: 16px;\n    height: 750px;\n    margin: 5px;\n    background-color: #cccccc;
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/stylesheet.css b/stylesheet.css
--- a/stylesheet.css	(revision 288fc5e9ef932500bab55ab4661e2cc23ada6f4a)
+++ b/stylesheet.css	(date 1633953255991)
@@ -1,3 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap');
+
+body {
+    font-family: 'Quicksand', sans-serif;
+}
+
 button {
     display: inline-block;
     padding: 15px 25px;
@@ -10,7 +16,7 @@
     background-color: indianred;
     border: none;
     border-radius: 15px;
-    box-shadow: 0 9px #999;
+    box-shadow: 3px 9px #999;
 }
 
 button:hover {background-color: darkred}
@@ -20,6 +26,19 @@
     box-shadow: 0 5px #666;
     transform: translateY(4px);
 }
+
+button.blue {
+    background-color: #393eb9;
+}
+
+button.blue:hover {background-color: darkblue}
+
+button.green {
+    background-color: #96cd5c;
+}
+
+button.green:hover {background-color: darkgreen}
+
 .container {
     display: grid;
     grid-template-columns: 400px 600px 400px;
@@ -31,4 +50,5 @@
     padding: 16px;
     height: 750px;
     margin: 5px;
-    background-color: #cccccc;
\ No newline at end of file
+    background-color: #cccccc;
+}
\ No newline at end of file
Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># cookie_clicker\njs game\n\nhttps://n-kaplan.github.io/cookie_clicker/\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision 288fc5e9ef932500bab55ab4661e2cc23ada6f4a)
+++ b/README.md	(date 1633953255989)
@@ -1,4 +1,19 @@
-# cookie_clicker
-js game
+
+<h1>Cookie Clicker Game</h1>
 
-https://n-kaplan.github.io/cookie_clicker/
+<a href ="https://n-kaplan.github.io/cookie_clicker/">cookie clicker</a>
+
+<p align="center">
+  <a href="#creators">Creators</a> •
+  <a href="#purpose">Purpose</a> •
+  <a href="#project-page">Project page</a>
+</p>
+
+## Creators
+* <a href="https://github.com/AnaisKabambi">A. Kabambi</a><br>
+* <a href="https://github.com/N-Kaplan">N. Kaplan</a>
+
+## Purpose
+A collaboration implementing recently acquired skills in Javascript, HTML and CSS.
+
+## Project Page
\ No newline at end of file
